CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Timestamping can now be enabled using the `timestamp` command line argument to the ASB.
+  JSON logs and ASB output in an interactive terminal are no longer timestamped by default.
```

Comment:
I believe this was changed to be the default again with https://github.com/comit-network/xmr-btc-swap/pull/765/commits/9c9091ea7f7d3d167a179c8f85a3d5448600bf54.

Reply:
Ahh thanks for that pick